simplified-cardcreator/src/main/java/org/nypl/simplified/cardcreator/CardCreatorActivity.kt

```diff
@@ -16,5 +17,14 @@ class CardCreatorActivity : AppCompatActivity() {
     super.onCreate(savedInstanceState)
     binding = ActivityCardCreatorBinding.inflate(layoutInflater)
     setContentView(binding.root)
+    if (intent.extras.getBoolean("isLoggedIn")) {
+      Cache(this).isJuvenileCard = true
+      binding.toolbarTitleTv.text = getString(R.string.create_child_card)
+    }
+  }
+
+  override fun onDestroy() {
+    super.onDestroy()
+    Cache(this).clear()
```

Comment:
Does this introduce a lifecycle issue? What happens if the user rotates their device and destroys/recreates the activity? I'm not sure if that's actually a problem for your cache or not.